### ISSUE TITLE
expat has boggus headers dir CMake config

### ISSUE THREAD

**Issue by mmuman** [haikuports/haikuports#9814]
Trying to update OpenColorIO, I got an error about non existing directory `/boot/system/lib/include` when importing expat.

I traced it to expat's `cmake/autotools/expat.cmake` line 62:
```
  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include"
```

It seems this is passed through autotools so we'd need to add some `@FOO@` substitution probably.

**Comment by mmuman** [comment]
I can add a …__haiku.cmake but I'd need to detect if subarch and use headers/ or headers/xx

**Comment by mmuman** [comment]
oh ok `CMAKE_HAIKU_SECONDARY_ARCH_SUBDIR`

**Comment by mmuman** [comment]
Ok, almost have it fixed, except I'm not sure what to do with the [IMPORTED_LOCATION](https://cmake.org/cmake/help/latest/prop_tgt/IMPORTED_LOCATION.html) for the noconfig file… shouldn't it point to `develop/lib` instead of `lib`?

**Comment by mmuman** [comment]
Should be fixed in 654cc24. Still not sure for the libs though, but it should still work.
